hooks/post-up

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env sh
```

Comment:
We are consistently using `/bin/sh` as our `sh` path. I haven't seen anything noting that the `sh` shell should live elsewhere.